### ISSUE TITLE
Warninng!!! locomotion.examples.a1_robot_exercise could damage your motor!

### ISSUE THREAD

**Issue by YilangLiu** [yxyang/locomotion_simulation#7]
Hi, I was trying to run the `python -m locomotion.examples.a1_robot_exercise` to send open loop control. I was able to build package successfully and use Unitree SDK 3.2. However, once I hang up the robot and press the enter, the robot immediately switched to the initial pose in like 0.5 seconds. From my terminal window, the code consistently output "failure".
Then the motor started to smoke after running for 5 seconds. I was concerned if this code had any sort of safety implementation that could prevent motor from being damaged?

**Comment by yxyang** [comment]
Hi Yilang,

Since I'm primarily working with Go1 right now, I am not familiar with the latest setup procedure of A1. Here are a few starting points if you want to investigate:

1. Does the low-level control example from Unitree's official SDK work? (I remember it's called `example_position`)?

2. Have you entered joint-damping mode and disabled the built-in Unitree controllers? Sometimes that can cause interference.

3. Upon reset, the robot is supposed to linearly interpolate from its starting motor position to the default reset position. When you say the robot "immediately" switch to the initial pose, it looks like this interpolation is not working. The relevant code is [here](https://github.com/yxyang/locomotion_simulation/blob/master/locomotion/robots/a1_robot.py#L289).

4. Lastly, my lateset A1 repo is [fast_and_efficient](https://github.com/yxyang/fast_and_efficient), which is refactored from this. It should be much easier to work with.

**Comment by Charlescai123** [comment]
> Hi, I was trying to run the `python -m locomotion.examples.a1_robot_exercise` to send open loop control. I was able to build package successfully and use Unitree SDK 3.2. However, once I hang up the robot and press the enter, the robot immediately switched to the initial pose in like 0.5 seconds. From my terminal window, the code consistently output "failure". Then the motor started to smoke after running for 5 seconds. I was concerned if this code had any sort of safety implementation that could prevent motor from being damaged?

Exactly!!! I also tried this on my A1 several weeks ago and the motor began smoke afterwards... It's the right front leg and the calf motor is burnt. Originally I was thinking that this is due to the motor itself. Currently it seems it has something to do with the code rather than the motor...
